.circleci/config.yml

```diff
@@ -76,7 +76,7 @@ jobs:
             composer install
             composer global require "phpunit/phpunit=5.7.*"
             rm -rf $WP_TESTS_DIR $WP_CORE_DIR
-            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
+            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 5.8.1
```

Comment:
`5.8.2` [introduces some tricky changes](https://make.wordpress.org/core/2021/09/27/changes-to-the-wordpress-core-php-test-suite/) to the test framework

Reply:
Nice temporary fix!